packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/cloudwatch-logs.ts

```diff
@@ -74,14 +74,12 @@ export class CloudWatchLogsExtension extends ServiceExtension {
     }
   }
 
-  public prehook(service: Service, scope: cdk.Construct) {
+  public prehook(service: Service, scope: Construct) {
     this.parentService = service;
 
     if (!this.logGroup) {
-      // If the user did not manually pass a log group in create one
-      this.logGroup = new awslogs.LogGroup(scope, `${service.id}-logs`, {
-        logGroupName: `${service.id}-logs`,
```

Comment:
Not sure if this would be considered a breaking change? Since we're changing the default value of the logGroupName from what users might currently be expecting to an autogenerated cfn default. Is there a particular reason why we can't keep this logic?

Reply:
This is a breaking change relative to the feature branch, but the feature branch is not merged into the main branch yet, so it won't be a breaking change for customers. I discovered that this was causing an issue where stacks could not be deleted and redeployed because a retained log group had a name collision. So I'm fixing it as part of this merge to the feature branch.

Reply:
Cool, it's something we should keep in mind when we're ready to merge into master